lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -106,6 +116,7 @@ def etl_presence_absence(*, action: str):
                         additional_details = sample_details(received_sample))
 
                     received_sample_id = str(received_sample["sampleId"])
+                    chip = received_sample["chip"]
```

Comment:
For extra robustness, how about an `assert chip` so that it must have a truthy value?